modules/update-availability-faq.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+[id="channel-differences_{context}"]
+*What are the differences between each of the update channels?*
+
+* A new release is initially added to the `candidate` channel and is often available as an update target for clusters on the `candidate` channel.
+
+* After successful internal and external testing, a release on the `candidate` channel is supported by being promoted to the `fast` channel, and is often available to clusters on the `fast` channel.
+
+* After a delay and more successful testing, a release on the `fast` channel is finally promoted to the `stable` channel and is often available to clusters on the `stable` channel. This delay represents the only difference between the `fast` and `stable` channels.
```

Comment:
Can we drop "and more successful testing" here?  Red Hat QE has wrapped up their internal testing before we declare support for the release and promote it to fast  channels.  Maybe external customers are testing during the delay between fast and stable.  Both internal and external users in fast channels are updating to the new release, and they may hit bugs, and report those bugs, and perhaps those bugs result in declared update risks, but that's just "operating a supported cluster", not "testing".